config/jobs/update-drivers-website/update-drivers-website.yaml

```diff
@@ -0,0 +1,23 @@
+periodics:
+  - name: update-drivers-website
+    cron: '0 0 * * *'  # every midnight, daily
```

Comment:
I think this is fine. Just as a note: it means the index and website can have a delay of 23:59:59:... Am I missing something?

If I'm right, we could then find a way to trigger this when the bucket content changes (aka when new artifacts are actually published).

@Issif @FedeDP WDYT?

Reply:
We discussed with @FedeDP about that point. We can't trigger the update after each change:
* it will trigger the job a lot of times when several drivers are changed
* we need to wait til the drivers are built and pushed, or it's useless
* the trigger of the jobs to build the drivers doesn't ensure the drivers will be on S3
So we decided the best way is to do a periodic update, we could increase the frequency to be sure.

Reply:
Totally agree @Issif. Thanks